recipes/tsl-robin-map/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+import os
+from conans import ConanFile, CMake
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        test_package = os.path.join("bin", "test_package")
+        self.run(test_package, run_environment=True)
```

Comment:
```suggestion
        if not tools.cross_building(self.settings):
            test_package = os.path.join("bin", "test_package")
            self.run(test_package, run_environment=True)
```

Reply:
Thanks for pointing this out, I've updated the recipe.